### DISCUSSION TITLE
Newlines in response from LLM

### DISCUSSION THREAD

**Discussion by tricky98** [open-webui/open-webui#204] [Q&A] [answered]
Is there a way to get newlines sent by the LLM to be displayed correctly in the WebUI?
I have been trying to get code from several of the LLM's, and it is all showing as a single line in the browser.
When I do an export of the conversation, I can see the model is sending `\n`, but this is being ignored by the browser.
How would I convert these to `<br/>` instead?
Thanks

**Comment by tjbck** [comment]
Hi, this issue has been addressed and you should update to the latest version if you're using safari. Thanks!

**Reply by tricky98** [reply]
Thanks for the prompt reply!
I pulled the docker image yesterday (v1.0.0-alpha.21) -- should I re-pull or build out myself?

**Reply by tjbck** [reply, marked as answer]
The webui is being constantly updated, so yes, you should pull to get the latest version :)

**Reply by tricky98** [reply]
Thanks.  Working now.
